s3torchconnector/src/s3torchconnector/s3iterable_dataset.py

```diff
@@ -68,13 +79,15 @@ def from_prefix(
         s3_uri: str,
         *,
         region: str,
+        endpoint: str = "",
```

Comment:
Same here for default `None`

Reply:
Changed the default value of endpoint parameter to None instead of empty string